lib/go/Makefile

```diff
@@ -21,7 +21,7 @@ export GOPATH
 
 # Only set PROTOC_VER if it has an empty value.
 ifeq (,$(strip $(PROTOC_VER)))
-PROTOC_VER := 3.9.1
+PROTOC_VER := 24.3
```

Comment:
Per @bswartz where does this version number come from?

Reply:
This is just the latest version from https://github.com/protocolbuffers/protobuf/releases when I open this PR. I will update it again.